test/test-functions

```diff
@@ -99,11 +103,20 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    UUID=$(blkid -o value -s UUID "${LOOPDEV}p1")
```

Comment:
Is this really necessary? If suse supports booting from /dev/sda1, we should continue to do that. All those distro-specific paths should be avoided unless unavoidable.